modules/spring-data-ext/README.txt

```diff
@@ -1,14 +1,29 @@
 Apache Ignite Spring Module
 ---------------------------
 
-Apache Ignite Spring Data extension provides an integration with Spring Data framework.
+Apache Ignite Spring Data 2.2 extension provides an integration with Spring Data 2.2 framework.
 
-Importing Spring Data Module In Maven Project
+Main features:
+
+- Supports multiple Ignite instances on same JVM (@RepositoryConfig).
+- Supports query tuning parameters in @Query annotation
+- Supports projections
+- Supports Page and Stream responses
+- Supports Sql Fields Query resultset transformation into the domain entity
+- Supports named parameters (:myParam) into SQL queries, declared using @Param("myParam")
+- Supports advanced parameter binding and SpEL expressions into SQL queries:
+- Template variables:
+    - #entityName - the simple class name of the domain entity
+- Method parameter expressions: Parameters are exposed for indexed access ([0] is the first query method's param) or via the name declared using @Param. The actual SpEL expression binding is triggered by ?#. Example: ?#{[0] or ?#{#myParamName}
+- Advanced SpEL expressions: While advanced parameter binding is a very useful feature, the real power of SpEL stems from the fact, that the expressions can refer to framework abstractions or other application components through SpEL EvaluationContext extension model.
+- Supports SpEL expressions into Text queries (TextQuery).
+
+Importing Spring Data 2.2 extension In Maven Project
 ----------------------------------------
 
-If you are using Maven to manage dependencies of your project, you can add Spring Data extension
-dependency like this (replace '${ignite-spring-data-ext.version}' with actual version of Ignite Spring Data extension
-you are interested in):
+If you are using Maven to manage dependencies of your project, you can add Spring Data 2.2 extension
+dependency like this (replace '${ignite-spring-data-2.2-ext.version}' with actual version of Ignite Spring Data 2.2
```

Comment:
Why `2.2`? It seems the module was renamed to `ignite-spring-data-ext`

Reply:
Yes, I renamed ignite-spring-data-2.2-ext to ignite-spring-data-ext, as was suggested in the mailing list discussion https://lists.apache.org/thread/b03bny7xj9x5ty371srtdn9ysd72qht7 , but forgot to remove the '2.2' suffix in a couple of places.

Removed it now.